Wrong path on analytics.d.ts for Angular CLI: 8.0.0-beta.8

<!--🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅

Oh hi there! 😄

To expedite issue processing please search open and closed issues before submitting a new one.
Existing issues often contain information about workarounds, resolution, or progress updates.

🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅-->


# 🐞 Bug report

### Command (mark with an `x`)
<!-- Can you pin-point the command or commands that are effected by this bug? -->
<!-- ✍️edit: -->
```
- [ x ] new
- [ ] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

## 🔬 Minimal Reproduction

Run `ng new demo --collection=@angular/bazel --defaults`
<!--
Simple steps to reproduce this bug.

Please include: commands run (including args), packages added, related code changes.

If reproduction steps are not enough for reproduction of your issue, please create a minimal GitHub repository with the reproduction of the issue. 
A good way to make a minimal reproduction is to create a new app via `ng new repro-app` and add the minimum possible code to show the problem. 
Share the link to the repo below along with step-by-step instructions to reproduce the problem, as well as expected and actual behavior.

Issues that don't have enough info and can't be reproduced will be closed.

You can read more about issue submission guidelines here: https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-submitting-an-issue
-->

## 🔥 Exception or Error
<pre><code>
> sauce-connect-launcher@1.2.4 postinstall /home/vinh/code/lab/demo/node_modules/sauce-connect-launcher
> node scripts/install.js || nodejs scripts/install.js


> demo@0.0.0 postinstall /home/vinh/code/lab/demo
> ngc -p ./angular-metadata.tsconfig.json

node_modules/@angular/cli/models/analytics.d.ts(9,31): error TS2307: Cannot find module '../../../angular_devkit/core/src/analytics'.

</code></pre>


## 🌍 Your Environment
__@angular/bazel@8.0.0-beta.9__
__@angular/cli@8.0.0-beta.8__

<pre><code>
Angular CLI: 8.0.0-beta.8
Node: 10.15.1
OS: linux x64
Angular: 8.0.0-beta.9
... animations, bazel, common, compiler, compiler-cli, core
... forms, language-service, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.800.0-beta.8
@angular-devkit/build-angular     0.800.0-beta.8
@angular-devkit/build-optimizer   0.800.0-beta.8
@angular-devkit/build-webpack     0.800.0-beta.8
@angular-devkit/core              8.0.0-beta.8
@angular-devkit/schematics        7.3.6
@angular/cli                      8.0.0-beta.8
@ngtools/webpack                  8.0.0-beta.8
@schematics/angular               7.3.6
@schematics/update                0.800.0-beta.8
rxjs                              6.4.0
typescript                        3.3.4000
webpack                           4.29.6
</code></pre>

**Anything else relevant?**
<!-- ✍️Is this a browser specific issue? If so, please specify the browser and version. -->

<!-- ✍️Do any of these matter: operating system, IDE, package manager, HTTP server, ...? If so, please mention it below. -->

--- a/packages/angular/cli/models/analytics.ts
+++ b/packages/angular/cli/models/analytics.ts
@@ -13,7 +13,6 @@ import * as inquirer from 'inquirer';
 import * as os from 'os';
 import * as ua from 'universal-analytics';
 import { v4 as uuidV4 } from 'uuid';
-import { TimingOptions } from '../../../angular_devkit/core/src/analytics';
 import { getWorkspace, getWorkspaceRaw } from '../utilities/config';
 
 const analyticsDebug = debug('ng:analytics');  // Generate analytics, including settings and users.
@@ -309,7 +308,7 @@ export class UniversalAnalytics implements analytics.Analytics {
     this._dirty = true;
     this._ua.pageview({ dp, dh, dt, ...vars });
   }
-  timing(utc: string, utv: string, utt: string | number, options: TimingOptions = {}) {
+  timing(utc: string, utv: string, utt: string | number, options: analytics.TimingOptions = {}) {
     const vars = this._customVariables(options);
     analyticsLogDebug('timing utc=%j, utv=%j, utl=%j, %j', utc, utv, utt, vars);
 

